### ISSUE TITLE
Hardcoded extensions are still compressed

### ISSUE THREAD

**Issue by Weissrolf** [cfbao/CryptSync#1]
#Hallo,

after two years I finally managed to build my own version out of your changes, being no programmer. Unfortunately it turned out that your version still compresses those hardcoded extensions that are meant to be copied and ecnrypted uncompressed. I tested this with JPG files.

It seems that the "std::wstring orig_ext = CPathUtils::GetFileExtension(orig);" + IF part does not work to single out those extensions. So "compression = 0" does not seem to be applied, but rather the default compression level (=3 in your case).

Since I would like to add more extensions to the list (raw image files) I would be happy about any hint on how to get this to work properly?!

Thanks and regards.

**Comment by cfbao** [comment]
I saw your comment here: https://github.com/stefankueng/CryptSync/issues/6#issuecomment-538327845

I'm pretty sure the reason you still see compression is because your files' extensions are in uppercase, whereas the hard-coded exclusions are in lowercase.

**Comment by Weissrolf** [comment]
Yes, upper vs lower case was the solution. Thanks for the quick hint!
